pythreepio/pythreepio/utils.py

```diff
@@ -1,11 +1,15 @@
+#importlib.resources is imported  to provide access to resources within packages
 try:
     import importlib.resources as pkg_resources
+#for easier accessibility  importlib_resources can be refered to as pkg_resources
 except ImportError:
     import importlib_resources as pkg_resources
 import json
 import static
+#static is imported to Serve static or templated content via WSGI 
```

Comment:
For comments, we adhere to [flake8](https://flake8.pycqa.org/en/latest/) which requires there be a space after a comment. Could you update the comments in this file to have a space after #?

accepted:
`# This is a comment`

not accepted:
`#this is a comment`